object-construction-checker/tests/socket/ACSocketTest.java

```diff
@@ -333,72 +320,6 @@ void makePrependableSocket() throws IOException {
         final PrependableSocket prependableSocket = new PrependableSocket(null);
     }
 
-
-
-
-
-//    private void acceptConnections() {
```

Comment:
why is all this commented out?

Reply:
still have the same question

Reply:
It was commented out, I just removed it because it contains some methods/fields that are not defined in this class, so we can't test it. I added it again https://github.com/kelloggm/object-construction-checker/pull/291/commits/03e52aad72ffba8808e8d6ef71189925d887c288